core/primitives-core/src/profile.rs

```diff
@@ -232,6 +226,6 @@ mod test {
     #[test]
     fn test_profile_data_debug() {
         #[cfg(not(feature = "costs_counting"))]
-        println!("{:#?}", ProfileData::default());
+        println!("{:#?}", ProfileData::new_enabled());
```

Comment:
This is just to check that we don't panic, right? I guess we might as well remove the ` #[cfg(not(feature = "costs_counting"))]` on the previous line then?

Reply:
Yep.  I'll remove it.